### DISCUSSION TITLE
How to use Frigate's stream inside the Homekit Bridge (to reduce connections to the camera?)

### DISCUSSION THREAD

**Discussion by chench00** [blakeblackshear/frigate#6552] [General]
I am looking to pull my cameras streams inside the Homekit Bridge integration for easier viewing but after configuring the Restream option in order to reduce connections to the camera, I am not so sure on the best way to achieve this. I thought about using `rtsp://<frigate_host>:8554/<camera_name>` as described [here](https://docs.frigate.video/configuration/restream#reduce-connections-to-camera) but it simply fails (which I am not sure why).

Here's a sample of my go2rtc configuration:

        go2rtc:
          streams:
            driveway_cam:
              - rtsp://{FRIGATE_RTSP_USER}:{FRIGATE_RTSP_PASSWORD}@192.168.141.14:554/cam/realmonitor?channel=1&subtype=0
            backyard_cam:
              - rtsp://{FRIGATE_RTSP_USER}:{FRIGATE_RTSP_PASSWORD}@192.168.141.15:554/cam/realmonitor?channel=1&subtype=0
            living_room_cam:
              - rtsp://{FRIGATE_RTSP_USER}:{FRIGATE_RTSP_PASSWORD}@192.168.141.16:554/cam/realmonitor?channel=1&subtype=0

        ##########################################################
        ## Driveway
        ##########################################################

        cameras:
          driveway_cam:
            ffmpeg:
              output_args:
                record: preset-record-generic-audio-copy
              inputs:
                - path: rtsp://127.0.0.1:8554/driveway_cam # <--- the name here must match the name of the camera in restream
                  input_args: preset-rtsp-restream
                  roles:
                    - record
                    - detect

Thank you.

**Comment by NickM-27** [comment]
How are you running frigate? The stream most likely fails because port 8554 is not mapped / enabled

**Reply by chench00** [reply]
@NickM-27 That was it. The port wasn't mapped in docker-compose file. Would this be the best URL to use? Mainly looking to keep connections to the camera at a minimum.

**Reply by NickM-27** [reply]
yes that would be the suggested URL
